[SQUAD 2] uploading Slack user guide (#17)
tryber/hackatrybe-covid: Conhecendo_e_usando_o_Slack.md

# Conhecendo e usando o Slack

##### O que é o Slack?

O Slack é uma plataforma para comunicação de todo tipo de organização. Muito versátil, possui canais onde discutem-se assuntos acerca de algum tema. Podemos enviar mensagens de texto e voz assim como documentos e videos. Tendo impecável organização, o Slack é um sucesso!

##### Por que usar o Slack?

> Com uma interface muito bem desenvolvida é fácil e gostoso usar o Slack para comunicar com a sua equipe, como ele centraliza as mensagens fica facil achar aquele arquivo antigo que voce precisa no momento. Com disponibilidade mobil, voce pode baixar o app para seu celular e levar todas as conversas no seu bolso!

Reviewer: ```suggestion
Com uma interface muito bem desenvolvida é fácil usar o Slack para comunicar com a sua equipe. Como ele centraliza as mensagens, fica fácil achar aquele arquivo ou conversa antiga que voce precisa de no momento. Com disponibilidade mobile, voce pode baixar o app para seu celular e levar todas as conversas no seu bolso!

```

> Com uma interface muito bem desenvolvida é fácil e gostoso usar o Slack para comunicar com a sua equipe, como ele centraliza as mensagens fica facil achar aquele arquivo antigo que voce precisa no momento. Com disponibilidade mobil, voce pode baixar o app para seu celular e levar todas as conversas no seu bolso!

Reviewer 2: ```suggestion
Com uma interface muito bem desenvolvida é fácil usar o Slack para comunicar com a sua equipe. Como ele centraliza as mensagens, fica fácil achar aquele arquivo ou conversa antigo que você precisa no momento. Com disponibilidade mobile, você pode baixar o app para seu celular e levar todas as conversas no seu bolso!
```

> O slack usa de notificações pra você não ficar de fora das novidades que chegam nos canais ou mensagens dirigidas somente à você. Isso aumenta muito a produtividade, aumenta muito a eficiecia da comunicação. Te aconselho a experimenta-lo e tenho certeza que você vai gostar.

Reviewer: ```suggestion
O Slack usa notificações pra você não ficar de fora das novidades que chegam nos canais ou mensagens dirigidas somente à você. Isso aumenta muito a produtividade e a eficiência da comunicação. 
```

> O slack usa de notificações pra você não ficar de fora das novidades que chegam nos canais ou mensagens dirigidas somente à você. Isso aumenta muito a produtividade, aumenta muito a eficiecia da comunicação. Te aconselho a experimenta-lo e tenho certeza que você vai gostar.

Reviewer 2: ```suggestion
O slack usa notificações pra você não ficar de fora das novidades que chegam nos canais ou mensagens dirigidas somente a você. Isso aumenta muito a produtividade e a eficiência da comunicação.
```

### Atenção!

> - O Slack não tem aplicativo para Windows Phone

Reviewer: ```suggestion
### Atenção!

- O Slack não tem aplicativo para Windows Phone
```

## Boas praticas

- Perfil pessoal:

> Iniciando o uso do Slack, preencha seu perfil pessoal com nome completo e numero de telefone. Voce deve colocar uma foto do seu rosto centralizado para facilitar a identificação. Preencha a descrição da sua ocupação com um texto objetivo que tenha clareza no campo "o que eu faço". Você tambem pode configurar um nome de exibição!

Reviewer 2: ```suggestion
Iniciando o uso do Slack, preencha seu perfil pessoal com nome completo e número de telefone. É aconselhável colocar uma foto do seu rosto centralizado para facilitar a identificação. Preencha a descrição da sua ocupação com um texto objetivo que tenha clareza no campo "o que eu faço". Você também pode configurar um nome de exibição!
```

[![N|Solid](./perfil_slack.png)]

- Atente-se a:

> Sempre leia as conversas que já constam nos seus canais, evite repetir questões já trabalhadas. Os canais do Slack são como grupos de pessoas que tem um objetivo em comum.

Reviewer 2: ```suggestion
Sempre leia as conversas que já constam nos seus canais, evite repetir questões já trabalhadas. Os canais do Slack são como grupos de pessoas que têm um objetivo em comum.
```

- O que são os canais?

> Voce pode adicionar os canais de seu interesse e acompanhar a comunicação da sua instituição. O Slack também conta com mensagens diretas para você alcançar alguem do seu time rapidamente.

Reviewer 2: ```suggestion
Você pode adicionar os canais de seu interesse e acompanhar a comunicação da sua instituição. O Slack também conta com mensagens diretas para você alcançar alguém do seu time rapidamente.
```

- Atente-se a:

> Não fugir do assunto de um canal postando coisas que não pertencem ao proposito do mesmo.

Reviewer 2: ```suggestion
Não fugir do assunto de um canal postando coisas que não pertencem ao proposito dele.
```

- Use menções:

No Slack é possivel notificar alguem mencionando seu nome de exibição com um "@". Por ex: "@HugoJr"

- E claro:

> #### Divirta-se com essa incrível ferramenta!

Reviewer: ```suggestion
```

Uma vez que nosso foco é o estudo e o trabalho, acho que uma mensagem de `divirta-se` sai um pouco do tom. O que você acha?
